client/my-sites/customer-home/controller.jsx

```diff
@@ -67,7 +68,8 @@ export async function maybeRedirect( context, next ) {
 			checklist: launchpadChecklist,
 		} = await fetchLaunchpad( slug );
 
-		const shouldShowLaunchpad = true;
+		const isRemovedFlow = REMOVED_TAILORED_FLOWS.find( ( { flow } ) => flow === siteIntentOption );
+		const shouldShowLaunchpad = ! isRemovedFlow;
```

Comment:
I think it would be better to delegate determining if a flow is removed to a function inside the `flow-redirect-handler`, maybe we could have a function called isRemovedFlow( flow: string );

An implementation could be:
````
const isFlowRemoved = (siteIntentOption) => !!REMOVED_TAILORED_FLOWS.find(({ flow }) => flow === siteIntentOption);
````
We wouldn't need to export the ` REMOVED_TAILORED_FLOWS` since it can be limited to its domain and not need to be exposed.

Reply:
https://github.com/Automattic/wp-calypso/pull/95669/commits/550b909a3839ec08643a6b513e68e9d1efe26fb6